pkg/types/baremetal/validation/platform.go

```diff
@@ -283,9 +283,9 @@ func validateHostsCount(hosts []*baremetal.Host, installConfig *types.InstallCon
 func validateBootMode(hosts []*baremetal.Host, fldPath *field.Path) (errors field.ErrorList) {
 	for idx, host := range hosts {
 		switch host.BootMode {
-		case "", baremetal.UEFI, baremetal.Legacy:
+		case "", baremetal.UEFI, baremetal.Legacy, baremetal.UEFISecureBoot:
```

Comment:
Can we keep these in the same order as they are in the const declaration?
```suggestion
		case "", baremetal.UEFI, baremetal.UEFISecureBoot, baremetal.Legacy:
```